src/gui/color_clipboard.hpp

```diff
@@ -0,0 +1,45 @@
+//  SuperTux
+//  Copyright (C) 2024 bruhmoent
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#ifndef HEADER_SUPERTUX_GUI_COLOR_CLIPBOARD_HPP
+#define HEADER_SUPERTUX_GUI_COLOR_CLIPBOARD_HPP
+
+#include <memory>
+
+#include "video/color.hpp"
+
+class ColorClipboard 
```

Comment:
Does this need to be a class? The same thing could be done with a static variable under the `Color` class.

Reply:
Could be expanded on later.

Reply:
I'm not sure what other purpose a color clipboard could serve.